tools/run_tests/sanity/sanity_tests.yaml

```diff
@@ -24,4 +24,5 @@
 - script: tools/distrib/yapf_code.sh
 - script: tools/distrib/python/check_grpcio_tools.py
 - script: tools/distrib/check_shadow_boringssl_symbol_list.sh
+- script: tools/distrib/check_protobuf_pod_version.sh
   cpu_cost: 1000
```

Comment:
I think you need to reorder L27 and L28. Otherwise `cpu_cost: 1000` will be applied to `check_protobuf_pod_version.sh` instead of `check_shadow_boringssl_symbol_list.sh`

Reply:
The `cpu_cost` directive should actually be moved two lines above. The same thing happened when the last test case was checked in 😂